### ISSUE TITLE
Can't Install ATV Smart YouTube TV Bridge App

### ISSUE THREAD

**Issue by BahadirKaya90** [yuliskov/SmartTubeNext#867]
It is impossible to install ATV Smart YouTube TV Bridge app on the Android TVs because the signature of the app on TV and your app's doesn't match. I tried everything to make iy work but no success. I can't totally remove the official YouTube app since my TV isn't rooted. I tried adb commands and it seems uninstalled but the apps still in the app list on TV.

Can you sign your app with the official YouTube signature?

**Comment by ACR-Jeff** [comment]
SmartYoutubeTV is Outdated and not maintained any longer, If you can SmartTubeNext to your TV. I am not familiar with an actual Android TV and what it allows.

**Comment by BahadirKaya90** [comment]
I already know what you are telling. I don't talk about old Smart YouTube TV I'm talking about the extention of SmartTube Next which is called ATV Smart YouTube TV Bridge. This app makes it possible to open SmartTube Next application when we make a voice search on Android TV/Box and if the results are YouTube videos.

But we can't update the YouTube app since this app version is older than the installed YouTube. I tried to uninstall YouTube with ADB comments but it doesn't removed totally and still it is listed as installed. I tried to change the version code of the above app but this time ADB command returns "Signature verification is failed since the installed app and update has different signatures."

As a result I can't install this extension and when make a voice search on TV I can't open the videos on SmartTube Next. They are opened on official YouTube.

I hope I am a understandable.

**Comment by zitev** [comment]
The problem is the same with me, I see that it has not been solved since then.

**Comment by Ricky-Garcia1110** [comment]
hello, is there any solution for this? I'm the same with chromecast google tv.

**Comment by emranovin** [comment]
It seems you should uninstall the original Youtube app COMPLETELY. and thereby you need root you tv, sadly..

**Comment by shhahid7** [comment]
Yes, same issue with me, unable to install "ATV Smart YouTube TV Bridge App".

Currently, I'm using GoogleTV and I've installed SmartTube v18.50 and uninstalled Youtube TV using adb command. The Smarttube is succesfully installed and The original Youtube TV app is completely uninstalled as i can confirm it.
but as voice search isnt working its prompting me to install Youtube tv from google play store. I tried to install ATV Smart Youtube TV Bridge using adb or from direct app install option but its failed to install.

I dont know the reason but i am still looking for it. Requesting developer kindly share tip or trick to install ATV Bridge so that voice search can be work!
Thanks

**Comment by dislightdude** [comment]
I have a Xiaomi Mi Box S and I tried to delete the updates of the youtube app.
Doing that reverted my youtube app to version

**Comment by dislightdude** [comment]
I have a Xiaomi Mi Box S and I tried to delete the updates of the youtube app.
Doing that reverted my youtube app to version youtube.tv_2.10.17

The TV Smart YouTube TV Bridge seems to be same com.google.android.youtube.tv as the youtube app but it's version is 2.4 ( so the Bridge app has a smaller version than the youtube app from my Mi Box S )
Because of that when I try to update from ADB I get the downgrade error.

I think the developers should make a Bridge app with a version as same as the youtbe app ( 4.05.002 )
I don't know how to modify the version of the apk bridge app.
I tried to do that using a app on my phone but when I tried to install that app it told me that the app didn't have the same signature as the previously installed youtube app

**Comment by JGuilhermeCoelho** [comment]
I can't install ATV Smart YouTube TV Bridge on Philips TV with Google TV

**Comment by Arcami** [comment]
I'm having the same issue with a Xiaomi TV box second gen with Google TV

**Comment by dallna01** [comment]
Did the download link expire? I cant even find the apk for the bridge anywhere and the provided link doesnt work.
